src/query/procedure/module.hpp

```diff
@@ -117,6 +117,8 @@ class ModuleRegistry final {
   /// Returns the shared memory allocator used by modules
   utils::MemoryResource &GetSharedMemoryResource() noexcept;
 
+  bool RegisterMgProcedure(const std::string_view name, mgp_proc proc);
```

Comment:
```suggestion
  bool RegisterMgProcedure(std::string_view name, mgp_proc proc);
```
Clang-tidy didn't trigger on this?

Reply:
Don't know but good catch!